Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/App.xaml.cs

```diff
@@ -115,6 +106,20 @@ protected override void OnStart()
             }
         }
 
+        static IReadOnlyDictionary<string, string> GetAppSecretDictionary()
+        {
+            // If user has selected another Auth Type, override the secret dictionary accordingly.
+            var persistedAuthType = AuthTypeUtils.GetPersistedAuthType();
+            switch (persistedAuthType)
+            {
+                case AuthType.AAD:
+                    return AADAuthAppSecrets;
+                default:
+                    // Default to B2C.
```

Comment:
can self document default case by using fallthrough

```
case AuthType.B2C:
default:
```

No need for a comment that way.

Reply:
Good call, thanks